.eslintrc

```diff
@@ -0,0 +1,33 @@
+---
+
+root: true
+
+env:
+  browser: true
+  mocha: true
+  node: true
+
+rules:
+  comma-dangle: 0
+  curly: [2, multi-line]
+  eqeqeq: 2
+  indent: [2, tab, { SwitchCase: 0 }]
+  linebreak-style: [2, unix]
+  object-curly-spacing: [2, never]
+  semi: [2, always]
+  space-after-keywords: [2, always]
+  space-before-function-paren: [2, never]
+  spaced-comment: [2, always]
+  no-console: 0
+  no-irregular-whitespace: 2
```

Comment:
This line is unnecessary as it is included in `eslint:recommended`. (Regarding my previous, deleted comment, I simply had a typo in my config file...)